src/test/java/core/basesyntax/services/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String CORRECT_FILE_PATH = "src/test/resources/input.csv";
+    private static final String INCORRECT_FILE_PATH = "src/test/resources/wrongInput.csv";
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void read_TestCorrectInput() {
+        List<String> expected;
+        try {
+            expected = Files.readAllLines(Path.of(CORRECT_FILE_PATH));
```

Comment:
same here you can just use constant from FileWriterServiceImplTest

Reply:
Did the same here